docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1507 @@
+=======================
+Opflow Migration Guide
+=======================
+
+TL;DR
+-----
+The new :mod:`~qiskit.primitives`, in combination with the :mod:`~qiskit.quantum_info` module, have superseded
+functionality of :mod:`~qiskit.opflow`. Thus, the latter is being deprecated.
+
+.. note::
+
+    The use of :mod:`~qiskit.opflow` was tightly coupled to the :class:`~qiskit.utils.QuantumInstance` class, which
+    is also being deprecated. For more information on migrating the :class:`~qiskit.utils.QuantumInstance`, please
+    read the `quantum instance migration guide <http://qisk.it/qi_migration>`_.
+
+
+Background
+----------
+
+The :mod:`~qiskit.opflow` module was originally introduced as a layer between circuits and algorithms, a series of building blocks
+for quantum algorithms research and development.
+
+The recent release of the :mod:`qiskit.primitives` introduced a new paradigm for interacting with backends. Now, instead of
+preparing a circuit to execute with a ``backend.run()`` type of method, the algorithms can leverage the :class:`.Sampler` and
+:class:`.Estimator` primitives, send parametrized circuits and observables, and directly receive quasi-probability distributions or
+expectation values (respectively). This workflow simplifies considerably the pre-processing and post-processing steps
+that previously relied on this module; encouraging us to move away from :mod:`~qiskit.opflow`
+and find new paths for developing algorithms based on the :mod:`~qiskit.primitives` interface and
+the :mod:`~qiskit.quantum_info` module.
+
+This guide traverses the opflow submodules and provides either a direct alternative
+(i.e. using :mod:`~qiskit.quantum_info`), or an explanation of how to replace their functionality in algorithms.
+
+The function equivalency can be roughly summarized as follows:
+
+.. |operator_globals| replace:: ``operator_globals``
+.. _operator_globals: https://qiskit.org/documentation/apidoc/opflow.html#operator-globals/
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow Module
+     - Alternative
+   * - Operators (:class:`~qiskit.opflow.OperatorBase`, |operator_globals|_ ,
+       :mod:`~qiskit.opflow.primitive_ops`,
+       :mod:`~qiskit.opflow.list_ops`\)
+     - :mod:`qiskit.quantum_info` *Operators*
```

Comment:
If you add a label before the section you want to refer to, you can link to the section directly, see: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations

In this case, you can add `.. _quantum_info_operators:` in front of the `Operators` section of the `quantum_info` module level API.